src/subplots.ts

```diff
@@ -1489,19 +1490,20 @@ export class BasePlot extends PlotData {
 
   public viewPoint: Vertex = new Vertex(0, 0);
   public fixedObjects: any[] = [];
-  public movingObjects: newShape[] = [];
+  public absoluteObjects: newShape[] = [];
+  public relativeObjects: newShape[] = [];
 
   public font: string = "sans-serif";
 
   protected initScale: Vertex = new Vertex(1, -1);
   private _axisStyle = new Map<string, any>([['strokeStyle', 'hsl(0, 0%, 31%)']]);
-  private nSamples: number;
+  public nSamples: number;
 
   readonly features: Map<string, any[]>;
   readonly MAX_PRINTED_NUMBERS = 16;
   readonly TRL_THRESHOLD = 20;
   constructor(
-    public data: any,
+    data: any,
```

Comment:
Why is this necessary to remove the public ?

Reply:
I think data is not an attribute of plot, i removed it from all plots. I cannot for PlotData object yet so it is still an instance attribute, but should not.It should just be used to instantiate plot and then thrown